ArduCopter/mode_guided.cpp

```diff
@@ -618,10 +618,7 @@ void ModeGuided::set_angle(const Quaternion &q, float climb_rate_cms_or_thrust,
     guided_angle_state.update_time_ms = millis();
 
     // log target
-    copter.Log_Write_GuidedTarget(guided_mode,
-                           Vector3f(guided_angle_state.roll_cd, guided_angle_state.pitch_cd, guided_angle_state.yaw_cd),
-                           false,
-                           Vector3f(0.0f, 0.0f, climb_rate_cms_or_thrust), Vector3f());
+    copter.Log_Write_Guided_Attitude_Target(guided_mode, guided_angle_state.roll_cd, guided_angle_state.pitch_cd, guided_angle_state.yaw_cd, 0, 0, 0, guided_angle_state.thrust);
```

Comment:
Like I said above we should add this in. I prefer for it to be separate from thrust (though it wastes some bytes) as it makes log analysis easier by having units attached to it.
```suggestion
    copter.Log_Write_Guided_Attitude_Target(guided_mode, guided_angle_state.roll_cd, guided_angle_state.pitch_cd, guided_angle_state.yaw_cd, 0, 0, 0, guided_angle_state.thrust, guided_angle_state.climb_rate_cms);
```

Reply:
Done!!